.rubocop_local.yml

```diff
@@ -0,0 +1,19 @@
+#
+# Overrides
+#
+GlobalVars:
+  AllowedVariables:
+  # Loggers
+  - $log
+  - $vim_log
+#
+# Special Exclusions
+#
+AllCops:
+  TargetRubyVersion: 2.2
+Metrics/LineLength:
```

Comment:
I feel like we shouldn't add any exclusions for Gemfile in gems since it shouldn't be used.